upgrade_order.md

```diff
@@ -1 +1 @@
-website > local_user_index > group
\ No newline at end of file
+user_index->local_user_index->group->website->user
```

Comment:
This isn't right... local user index removes the field and calls into users so it must go after users.

Reply:
And group c2c_join_group has a field removed so groups must go before local user index too.

Reply:
I'm pretty sure it should be user_index->group->user->local_user_index->website